src/hooks/useConnectPartyGoers.ts

```diff
@@ -8,6 +9,12 @@ import { useUserLastSeenThreshold } from "./useUserLastSeenThreshold";
 export const useConnectPartyGoers = () => {
   const lastSeenThreshold = useUserLastSeenThreshold();
 
+  useEffect(() => {
+    console.count("Created a listener to fetch data");
+  }, []);
+
+  console.log({ lastSeenThreshold });
```

Comment:
Dont forget to remove this

Reply:
Thank you. Strangely enough, Eslint didn't catch it